runtime/interpreter/value.go

```diff
@@ -9303,76 +9805,99 @@ func (v PathValue) Equal(other Value, _ *Interpreter, _ bool) bool {
 		otherPath.Domain == v.Domain
 }
 
+func (v PathValue) HashInput(_ *Interpreter, _ func() LocationRange, scratch []byte) []byte {
+	scratch[0] = HashInputTypePath
+	scratch[1] = byte(v.Domain)
```

Comment:
`v.Domain` is uint being cast to byte.  Can `v.Domain` ever be larger than 255?

Reply:
Good catch! The current domains fit into `byte`, but I changed the type of `PathDomain` to `byte` anyways in 34429a44bff217ed394a453f04bddb4b46225f0b